### ISSUE TITLE
logback.xml for es-core is packaged into the client jar

### ISSUE THREAD

**Issue by afremuth-figure** [provenance-io/event-stream#37]
## Summary of Bug

logback.xml is being leaked into the client jar.

```
21:12:48,727 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs multiple times on the classpath.
21:12:48,727 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [jar:file:/Users/user/.gradle/caches/modules-2/files-2.1/io.provenance.eventstream/es-core/0.5.0/929522cc21c21870eaecf5d0b4704f89739a276/es-core-0.5.0.jar!/logback.xml]
21:12:48,727 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [file:/Users/user/dev/service-loan-submission/service/build/resources/main/logback.xml]
```

## Version

0.5.0

## Steps to Reproduce

Use this lib in a project that uses logback itself.  The error should be seen on startup.

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by mtps** [comment]
Confirmed. logback.xml needs moved to cli subproject and references to logback in the build cleaned up. es-core should only reference kotlin logging or slf4j in its dependency tree. the logback jars are leaking in from somewhere else.

**Comment by wbaker-figure** [comment]
closed by #39
